src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.cs

```diff
@@ -130,6 +130,6 @@ public partial struct ComInterfaceDispatch
         /// Called when a request is made for a collection of objects to be released outside of normal object or COM interface lifetime.
         /// </summary>
         /// <param name="objects">Collection of objects to release.</param>
-        protected abstract void ReleaseObjects(IEnumerable objects);
+        protected internal abstract void ReleaseObjects(IEnumerable objects);
```

Comment:
Why is this needed?

Reply:
This is called by `TrackerObjectManager.ReleaseExternalObjectsFromCurrentThread` and `TrackerObjectManager` is no longer a nested class of `ComWrappers`.